test/EVMHost.cpp

```diff
@@ -59,7 +60,44 @@ evmc::VM& EVMHost::getVM(string const& _path)
 			cerr << endl;
 		}
 	}
-	return theVM;
+
+	if (vms.count(_path) > 0)
+		return vms[_path];
+
+	return NullVM;
+}
+
+bool EVMHost::checkVmPaths(std::vector<boost::filesystem::path> const& _vmPaths)
+{
+	bool evmVmFound = false;
+	bool ewasmVmFound = false;
+	for (auto const& path: _vmPaths)
+	{
+		evmc::VM& vm = EVMHost::getVM(path.string());
+		if (!vm)
+			return false;
+
+		if (vm.has_capability(EVMC_CAPABILITY_EVM1))
+		{
+			if (evmVmFound)
+			{
+				cerr << "Multiple evm1 evmc vms defined. Please only define one evm1 evmc vm." << endl;
```

Comment:
Looks a bit weird to first print the error to stderr and then throw using an assertion and not specify any error.
Would it work to just throw a regular exception and print the error string at the place where the exception in caught?

Reply:
i just remove the exception here. if something goes wrong the semantic tests will be disabled and the error is printed to stderr.

Reply:
now i throw an exception, that will be caught in `test/boostTest.cpp` and `test/tools/isoltest.cpp`. In both cases the exception message will just get printed and the program will terminate.